#!/usr/bin/env python
# -*- coding: utf-8 -*-
PACKAGE = "navig"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("timeout_device_silence", double_t, 0, "Таймаут, после которого считается, что устройство не отвечает [c]", 10)
gen.add("timeout_old_data", double_t, 0, "Таймаут, после которого данные считаются устаревшими [c]", 2)
gen.add("use_velocity_from_regul", bool_t, 0, "Флаг, использовать ли данные о скорости от регуляторов", False)
gen.add("delta_t", int_t, 0, "Период чтения сообщений из топика [Hz]", 10)

exit(gen.generate(PACKAGE, "navig", "LocalPositionEstimator"))